src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -230,12 +239,11 @@ function MultiSelect<T>({
     }, [items.map((item) => itemId(item)).join("|")]);
 
     React.useEffect(() => {
-        onSelection &&
-            onSelection({
-                newlySelected: selectedItems.slice(-1)[0],
-                createdItems: createdItems.current,
-                selectedItems,
-            });
+        onSelection?.({
```

Comment:
Haven't seen this before :) Can you send me please an article that this type of optional chaining is now used to test and fire a method?